vignettes/function-flow.Rmd

```diff
@@ -159,3 +159,129 @@ the data coverage period are dropped to end with a final diagnosis date
 using `get_final_diagnosis_date()`. For details on this censoring based
 on the periods with insufficient data coverage, see the
 `vignette("algorithm-logic")`.
+
+## Classifying the diabetes type {#classifying-diabetes-type}
+
+The second component of the OSDC algorithm classifies individuals from
+the extracted diabetes population as having either T1D or T2D. The
+output of this component is a `data.frame` that includes one row per
+individual in the diabetes population: one column with their PNR, two
+columns with inclusion dates (one "stable" date and one "raw" date - see
+the [Description of algorithm contents & logic](algorithm_logic.Rmd)
+vignette for an elaboration on what that entails), and one column with
+the diabetes type.
+
+<!-- TODO: Make sure this is the correct link - and add a link specific to the specific section where this is described -->
+
+This component will also have one user-facing function:
+`classify_diabetes_type`. This function takes the output of
+`extract_diabetes_population` as input.
+
+### Type 1 classification
+
+The classification of type 1 diabetes relies on the following two
+filters. To be classified as having type 1 diabetes, individuals must
+have:
+
+1.  At least one T1D primary diagnosis and only purchased insulins
+    (i.e., no purchases of any other type of GLDs), or
+2.  A majority of T1D primary diagnoses, purchased insulin within 180
+    days of diagnosis, and 2/3 of GLD doses are insulin.
+
```

Comment:
@signekb it's not really necessary considering the doc isn't big. Plus, links to other vignettes don't work when they get uploaded to CRAN. Unless we want this vignette to only be shown on the website rather than be viewable after installing the package itself.